### PR TITLE
docs(english): translating recent pt-br changes in docs to english

### PR DESCRIPTION
**Pull request by Nipodemos**
Hello!

Recently, in [this PR](https://github.com/jonataslaw/getx/pull/473), were added a few more documentation lines explaining more about dependency and how they work behind the scenes.

Since there was made only in pt-br, i decided to translate to english. I haven't added nor removed anything, just translated.
Thanks to @paulosilva159 for making the original version

### CODE REVIEW

**Comment by Nipodemos** [comment]
@jonataslaw i think you forgot me 😅

File: docs/en_US/dependency_management.md [hunk 1]
```diff
@@ -139,6 +139,45 @@ Get.putAsync<S>(
...
+
+  // optional: just like int`Get.put()`, it is for when you need to keep the
+  // instance alive thoughout the entire app. The difference is in Get.create
+  // permanent is true by default
+  bool permanent = true
+```
+
+### Diferences between methods:
+
+We will talk about the variables `_factory` and `_singl`. Both are essential in the process of creating
```
[the remark this answers is not in the document]
Comment (Nipodemos) [reply docs/en_US/dependency_management.md:165]: tá lá sim

ok, vou remover isso, aproveito e removo de lá também

File: docs/en_US/dependency_management.md [hunk 2]
```diff
@@ -139,6 +139,45 @@ Get.putAsync<S>(
...
+ and using our dependencies, because is by it that we can store, dele and recreate the instances.
+
+First, let's of the `fenix` of Get.lazyPut and the `permanent` of the other methods.
+
+The fundamental difference between `permanent` and `fenix` is how you want to store your instances.
+Reinforcing: by default, GetX deletes instances when they are not is use (let's say screen 1 has controller 1 and screen 2 has controller 2. When you move from screen 1 to screen 2, the controller 1 lost its use so it is erased), but if you want to opt to `permanent:true`, then the controller will not be lost in this transition - which is very usefult for services that you want to keep alive thoughout the entire application. `fenix` in the other hand is for services that you don't worry in losing between screen changes, but when you need that service, you expect that it is alive. So basically, it will dispose the unused controller/service/class, but when you need that, it will "recreate from the ashes" a new instance.
```
[the remark this answers is not in the document]
Comment (Nipodemos) [reply docs/en_US/dependency_management.md:171]: oh... eu podia jurar que era da forma que tá escrito... mas parando pra pensar, faz muito mais sentido desse jeito, já que a pessoa pode simplesmente voltar pra tela que ainda tá na stack, então tem se manter ativo

**Comment by jonataslaw** [comment]
LGTM

**Pull request merged**